test/index.test.js

```diff
@@ -80,6 +82,28 @@ describe('Sentry', function() {
         analytics.assert.deepEqual(window.RavenConfig.config, config);
       });
 
+      it('should allow and set custom versions', function() {
+        var config = {
+          logger: options.logger,
+          serverName: options.serverName,
+          whitelistUrls: options.whitelistUrls,
+          ignoreErrors: options.ignoreErrors,
+          ignoreUrls: options.ignoreUrls,
+          includePaths: options.includePaths,
+          maxMessageLength: options.maxMessageLength,
+          release: '2.4.0'
```

Comment:
Hmm, I actually think a better test would be to set this to `options.release` where the value is something other than `2.4.0`. Line 22 should default to `null` (normally when you declare your settings/options, make those align w whatever you will set as the default in your metadata). Then in this test before `initialize()` you would overwrite `sentry.options.customVersionProperty`.

that will test that this option works but also test that the fallback precedence is respected.